tests/test_cli.py

```diff
@@ -385,6 +386,7 @@ def _test_boolean_option(self, option_struct):
         cli = option_struct.cli_subcommand()
         _, config = cli.parse()
         option_value = config.get_config().get(option_with_underscores).value()
+        # Boolean values must always default to False
```

Comment:
why is this necessary?
The arg could use default=True

Reply:
This is a byproduct of how the `ConfigPrimitive(bool)` works. The field must be False by default and then it is set to True by setting it on the CLI (or in the config). You cannot have a default of True (and set it to False via the CLI/config)

Reply:
i see. okay